src/python/pants/backend/codegen/thrift/apache/rules.py

```diff
@@ -106,18 +107,21 @@ async def generate_apache_thrift_sources(
         *target_sources.snapshot.files,
     ]
 
-    result = await Get(
-        ProcessResult,
-        Process(
-            args,
-            input_digest=input_digest,
-            output_directories=(output_dir,),
-            description=f"Generating {request.lang_name} sources from {request.thrift_source_field.address}.",
-            level=LogLevel.DEBUG,
+    result = await fallible_to_exec_result_or_raise(
```

Comment:
Wasn't this re-named to `execute_process_or_raise`? If so, those need to be manually run. 

We can always just bulk run a migration at some point later when cleaning up the API.

Reply:
Ah yes, will change here and should probably fix the migrator somehow.